sphinx/util/osutil.py

```diff
@@ -175,13 +175,19 @@ def copyfile(source, dest):
 
 
 no_fn_re = re.compile(r'[^a-zA-Z0-9_-]')
+project_suffix_re = re.compile(' Documentation')
 
 
 def make_filename(string):
     # type: (str) -> unicode
     return no_fn_re.sub('', string) or 'sphinx'
 
 
+def make_filename_from_project(project):
```

Comment:
I don't know here is appropriate location of this function. But I placed it beside of `make_filename()`.